cabal.project

```diff
@@ -21,17 +21,24 @@ repository cardano-haskell-packages
 source-repository-package
   type: git
   location: https://github.com/IntersectMBO/formal-ledger-specifications.git
-  subdir: ledgerSrc/haskell/Ledger
-  tag: 79637b65e241537efbbad1a1125b7f7af8945b07
-  --sha256: sha256-L0oi+hFjFw8BE75jWtD+jAtBwaIDrMb8rqJGfBhSELE=
--- NOTE: If you would like to update the above, look for the `haskell-package`
--- branch in the `formal-ledger-specifications` repo where the HEAD contains
--- the generated code based on the most recent commit on `master`.
--- !IMPORTANT!: MAKE SURE TO TAG THE COMMIT IN `formal-ledger-specifications`
--- WHENEVER YOU UPDATE THIS SRP! Otherwise this could break, since each push
--- to `haskell-package` removes the old commits. With tagging, we are making
--- sure to keep the `formal-ledger-specifications` commits we are referencing,
--- otherwise they would be garbage collected by git.
+  subdir: generated
+  tag: d12f11db8ba86df1a20b5649292c95f254f1e40c
+  --sha256: 0dk9n4abxbdhd8w3zacvzh4dyxpr68n5iafhnv39yhb8hx6kwggs
+-- NOTE: If you would like to update the above, look for the `MAlonzo-code`
+-- branch in the `formal-ledger-specifications` repo and copy the SHA of
+-- the commit you need. The `MAlonzo-code` branch functions like an alternative
+-- `master / main` branch for the generated code, see the details here:
+-- https://github.com/IntersectMBO/formal-ledger-specifications/pull/530
+-- If you are working on something in `formal-ledger-specifications`
+-- and would like to see how they reflect here, just open a PR / draft PR
+-- in `formal-ledger-specifications` for your branch and that will
+-- automatically create a branch for the generated code that you can try here
```

Comment:
Is there a naming conventions for those branches that will be automatically created? It would be useful to mention it here so a person trying to use this workflow for the first time could find it easily

Reply:
In order to bring `master` to a functional state I'll merge this PR, but we can fixup this comment at some later point